### ISSUE TITLE
Livestream: Description area lost sidebar

### ISSUE THREAD

**Issue by infinite-persistence** [OdyseeTeam/odysee-frontend#905]
It doesn't affect me since I don't use it to scroll, but I assume it's needed for consistency and small set of users.

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/64950861/154494356-f0f82f61-fbc7-44e4-9166-24d876c0c41e.png">

**Comment by tzarebczan** [comment]
Good catch...usually don't run into longer ones. Rafael recent hid the double scroll, but yeah, we should handle this too.
